script.js

```diff
@@ -1 +1,44 @@
-// this is script file
+// all the dom elements are selected
+const addTodoInputElement = document.getElementById("addTodoInputField");
+const addTodoButton = document.getElementById("addTodoButton");
+const allTodoContainer = document.getElementById("todosContainer");
+
+// global variables
+let allTodosArray = [];
+
+// input data sanitization by using removing any tags
+const dataSanitization = (value) => {
+    return value.replace(/(<([^>]+)>)/gi, "");
+};
```

Comment:
Move this file to another file and import it from there.
e.g: 
```suggestion
// utility.js
export const sanitizeInput = (value) => {
    return value.replace(/(<([^>]+)>)/gi, "");
};

// script.js
import { sanitizeInput } from "./utility.js"
```